modules/common-transformer-stream/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/transformer/stream/common/parquet/ParquetSink.scala

```diff
@@ -50,14 +50,14 @@ object ParquetSink {
   ): Pipe[F, Transformed.Data, Unit] = { transformedData =>
     // As uri can use 's3a' schema, using methods from 'java.nio.file.Path' would require additional dependency responsible for adding appropriate 'java.nio.file.spi.FileSystemProvider', see e.g. https://github.com/carlspring/s3fs-nio/
     // Simple strings concat works for both cases: uri configured with and without trailing '/', bypassing usage of 'java.nio.file.Path'
-    val targetPath = s"${uri.toString}/${window.getDir}/${path.value}"
+    val targetPath = s"${resources.parquetOps.transformPath(uri.toString)}/${window.getDir}/${path.value}"
```

Comment:
Right, so we use `abfss...` like path to write parquet files. Not sure, but should we also use such 'Azure Data Lake like' URI in the shredding-complete message, so the loader use it in a COPY statements?